Projects/tailwind-clone/style.css

```diff
@@ -0,0 +1,30 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+* {
+    scrollbar-width: auto;
+    scrollbar-color: #052a42 #ffffff00;
+  }
+  
+  /* Chrome, Edge, and Safari */
+  *::-webkit-scrollbar {
```

Comment:
You don't need star selecter here!
It will work if you not use any * selecter llike

::webkit-scrollbar{
color: red;
}

Reply:
okk i will see on it